code/api/src/setup/database.js

```diff
@@ -27,8 +27,4 @@ connection
     console.error('ERROR - Unable to connect to the database:', err)
   })
 
-<<<<<<< HEAD
 export default connection
-=======
-export default connection
->>>>>>> remove comments
```

Comment:
Not sure why this is this here? Very strange

Reply:
Me neither, looks like this branch is getting rid of that comment. When I resolved conflicts yesterday I only did so by clicking the "accept incoming changes" option on VS-Code. So you'd think that would take care of these comments. 🤷